__tests__/healthcareProfessional.test.ts

```diff
@@ -0,0 +1,211 @@
+                                {
+                                    name: null,
+                                    locale: null
+                                }
+                            ]
+                        }
+                    ],
+                    spokenLanguages: [
+                        {
+                            nameNative: 'null',
+                            nameEn: 'null',
+                            iso639_3: 'null',
+                            nameJa: 'null'
+                        }
+                    ]
+                }
+            ],
+            isDeleted: null,
+            nameEn: null,
+            nameJa: null
+        }
+    }
+
+}
+
+const healthcareProfessionalQueryData = {
+    query: `mutation Mutation($input: HealthcareProfessionalInput) {
+        createHealthcareProfessional(input: $input) {
+          id
+          names {
+            lastName
+            firstName
+            middleName
+            locale
+          }
+          degrees {
+            nameJa
+            nameEn
+            abbreviation
+          }
+          spokenLanguages {
+            iso639_3
+            nameJa
+            nameEn
+            nameNative
+          }
+          specialties {
+            names {
+              name
+              locale
+            }
+          }
+          acceptedInsurance
+          isDeleted
+          createdDate
+          updatedDate
+        }
+      }`,
+    variables: {
+        input: {
+            acceptedInsurance: ['INSURANCE_NOT_ACCEPTED'],
+            degrees: [
+                {
+                    abbreviation: 'DG',
+                    nameEn: 'some degree EN',
+                    nameJa: 'some degree JA'
+                }
+            ],
+            names: [
+                {
+                    firstName: 'some first name',
+                    lastName: 'some last name',
+                    locale: 'ENGLISH',
+                    middleName: 'some middle name'
+                }
+            ],
+            specialties: [
+                {
+                    names: [
+                        {
+                            locale: 'ENGLISH',
+                            name: 'some specialty name'
+                        }
+                    ]
+                }
+            ],
+            spokenLanguages: [
+                {
+                    iso639_3: 'EN',
+                    nameEn: 'some spoken language EN',
+                    nameJa: 'some spoken language JA',
+                    nameNative: 'some spoken language NATIVE'
+                }
+            ],
+            facilityIds: facilityIds
+        }
+    }
+}
+
+describe('createHealthcareProfessional', () => {
+    let url: string
+
+    const server = new ApolloServer({
+        typeDefs: loadSchema(),
+        resolvers
+    })
+    
+    beforeAll(async () => {
+        ({ url } = await startStandaloneServer(server, {listen: { port: 0 }}))
```

Comment:
is port 0 ok here?

Reply:
Yup, port `0` was intentional so we don't have to worry about it interfering with dev or prod environments.